### ISSUE TITLE
Repo Transfer Proposal

### ISSUE THREAD

**Issue by amadeus** [pangloss/vim-javascript#416]
Since the owner @pangloss is no longer active in maintaining this repo and it is now, more or less, maintained by the community, I thought it could perhaps be a good idea (assuming @pangloss is ok with it, and same with the rest of the maintainers here) to create a new `vim-javascript` organization and have this repo transferred to it.

I looked into the docs regarding repo transfers and I can't really see much of a downside. All issues and PRs are transfered, references to this pangloss repo will automatically redirect, and it leaves us the maintainers in more direct control of it. In other words, nobody will have to update anything in their plugins, it should all 'just work'. More info can be read about transfers [here](https://help.github.com/articles/transferring-a-repository/).

I've already gone ahead and created the `vim-javascript` org just so no one could snatch it away from us in case we decide to do this.

Of course at the end of the day, it's up to @pangloss since really it's his baby.

**Comment by davidchambers** [comment]
If @pangloss is in favour of the proposal let's do it. I recently transferred a number of repositories to the @sanctuary-js organization and the move went smoothly. It's now much easier to add collaborators. The same benefit would apply here too (as you pointed out, @amadeus).

**Comment by bounceme** [comment]
I agree, with pangloss' approval. The past few months have seen few changes by pangloss, passing the repo over seems appropriate as there is a lot of new work that has been done, and as this is a somewhat active repo, convenient

**Comment by amadeus** [comment]
Sorry to bug, @pangloss any thoughts on this?

**Comment by qstrahl** [comment]
Late to the discussion, but I also think it would be good for the project to move to an officially community-owned organisation (pending @pangloss's approval). Pinging @pangloss again... will try to find a way to get in touch outside GitHub too.
